### PR TITLE
React Native Release 3.0.0

### PR DESCRIPTION
**Pull request by JiaoQian-01**
Things to consider before you submit the PR:

* [x] Has `CHANGELOG.md` been updated?
* [ ] Are tests passing locally?
* [x] Are the files formatted correctly?
* [ ] Did you test your change with either the sample apps that are included in the repository or with a blank app that uses your change?

## Description
Bumped the versions
Updated TestApp to use latest SDKs
Updated the CHANGELOG.md

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,17 +27,16 @@ App Center Auth is [retired](https://aka.ms/MBaaS-retirement-blog-post) and has
...
+* **[Behavior change]** Change the path to the minidump directory to use a subfolder in which the current contextual data (device information, etc.) is saved along with the .dmp file.
 
 #### iOS
 
-* **[Fix]** Fix an error where **appcenter.podspec.json** could not be found when using CocoaPods version 1.8.x.
-* **[Fix]** Fix issues with `use_frameworks!` directive.
-* **[Fix]** Fix an issue where React Native SDK would not send wrapperSdk information.
+* **[Improvement]** Update PLCrashReporter to 1.4.0.
 
 ___
```
Comment (JiaoQian-01) [inline CHANGELOG.md]: Please help me check if the content of CHANGELOG.ms is correct?

File: TestApp/ios/Podfile.lock [hunk 2]
```diff
@@ -249,11 +249,13 @@ PODS:
...
   - RNGestureHandler (1.5.0):
     - React
+  - RNScreens (2.0.0-alpha.34):
+    - React
   - Yoga (1.14.0)
 
 DEPENDENCIES:
-  - appcenter (from `../node_modules/appcenter/ios`)
   - appcenter-analytics (from `../node_modules/appcenter-analytics/ios`)
+  - appcenter-core (from `../node_modules/appcenter/ios`)
```
Comment (JiaoQian-01) [inline TestApp/ios/Podfile.lock]: appcenter is automatically changed to appcneter-core, is it correct?

Comment (annakocheshkova) [reply TestApp/ios/Podfile.lock]: Yes

File: CHANGELOG.md (hunk 1 above)
Comment (annakocheshkova) [reply CHANGELOG.md]: Looks correct

File: TestApp/ios/Podfile.lock [hunk 3]
```diff
@@ -401,6 +407,7 @@ SPEC CHECKSUMS:
   RNCAsyncStorage: 5ae4d57458804e99f73d427214442a6b10a53856
   RNFS: 795866388a01de4e470f3b1041a99cf20a343844
   RNGestureHandler: a4ddde1ffc6e590c8127b8b7eabfdade45475c74
+  RNScreens: 9d8a2900cd4f967af70dbfed74b64f77fbb6600a
```
Comment (JiaoQian-01) [inline TestApp/ios/Podfile.lock]: RNScreens was added automatically. Does it need to be deleted?

Comment (JiaoQian-01) [reply TestApp/ios/Podfile.lock]: @AnastasiaKubova please review it, Thanks.

File: TestApp/ios/Podfile.lock [hunk 4]
```diff
@@ -288,18 +290,20 @@ DEPENDENCIES:
...
   - RNFS (from `../node_modules/react-native-fs`)
   - RNGestureHandler (from `../node_modules/react-native-gesture-handler`)
+  - RNScreens (from `../node_modules/react-native-screens`)
   - Yoga (from `../node_modules/react-native/ReactCommon/yoga`)
 
 SPEC REPOS:
-  https://github.com/CocoaPods/Specs.git:
+  https://cdn.cocoapods.org/:
     - AppCenter
+  https://github.com/CocoaPods/Specs.git:
```
Comment (JiaoQian-01) [inline TestApp/ios/Podfile.lock]: This url was also changed automatically. Do I need to change it back?

File: TestApp/ios/Podfile.lock (hunk 3 above)
Comment (annakocheshkova) [reply TestApp/ios/Podfile.lock]: I think it's ok

File: TestApp/ios/Podfile.lock (hunk 4 above)
Comment (annakocheshkova) [reply TestApp/ios/Podfile.lock]: This is also ok

**Pull request merged**